### ISSUE TITLE
Entire Ubuntu and Chia 1.7 Freezing

### ISSUE THREAD

**Issue by laz954** [Chia-Network/chia-blockchain#6611]
While plotting Chia 1.7, either single plot or 3 plots in parallel after 4 hours the entire computer freezes, Chia as well as the Ubuntu system. Only way to recover is to hard reset the computer and start from scratch. Any work arounds ? Nvme is 1TB, i9 16 cores, 64G Ram. Latest Ubuntu, latest Chia release 1.7.  Does anyone have any recommendations and or suggestions to prevent the freezing ? Thanks..

**Comment by StrawberryOG** [comment]
Your Nvme could be getting too hot, or some other component in your system is overheating. If your Nvme also contains your OS that could also be another issue, you should be using a separate Nvme drive for temporary plotter for best results.

**Comment by Manoelrn007** [comment]
yes, it's very slow, almost stopping, there's something wrong with the debian/ubuntu linux version...

**Comment by manoeldrn** [comment]
Aplicação está muito lenta na distro debian/ubuntu e não tem relação com a configuração do hardware, estava usando no windows e funcionava muito bem.
